azure_client/azure_client.py

```diff
@@ -40,16 +40,17 @@ def create_uuid_container_in_azure(uuid):
     """Create uuid folder in the azure blob storage."""
     # TODO
     # ADD try-catch block
-    images_container = uuid + "-images"
-    audios_container = uuid + "-audios"
-    files_container = uuid + "-files"
-    videos_container = uuid + "-videos"
-
-    block_blob_service.create_container(images_container)
-    block_blob_service.create_container(audios_container)
-    block_blob_service.create_container(files_container)
-    block_blob_service.create_container(videos_container)
-    logger.debug("user folder creation successful")
+    if utility.verify_uuid(uuid):
```

Comment:
The UUID is already checked in `CreateUserFolder`, but is checked here again for unit testing.